hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -413,6 +447,22 @@ private void applyOperation(long counter) throws Exception {
         return null;
       });
       break;
+    case LIST_KEYS_LIGHT:
```

Comment:
I think it would be nice to have a case: `LIST_STATUS_LIGHT`, for the newer `listStatusLight` API as well!

Reply:
Done.